src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -51,41 +262,951 @@
+      boolean wasDeleted = RemoteStorageHelper.forceDelete(storage, BUCKET, 10, TimeUnit.SECONDS);
+      if (!wasDeleted && LOG.isLoggable(Level.WARNING)) {
+        LOG.log(Level.WARNING, "Deletion of bucket {0} timed out, bucket is not empty", BUCKET);
+      }
+    }
+  }
+
+  @Test
+  public void listAcceleratorTypesTest() {
+    AcceleratorTypeClient.ListAcceleratorTypesPagedResponse listAcceleratorTypes =
+        acceleratorTypeClient.listAcceleratorTypes(PROJECT_ZONE_NAME);
+    List<AcceleratorType> acceleratorTypes = Lists.newArrayList(listAcceleratorTypes.iterateAll());
+    assertThat(acceleratorTypes).isNotNull();
+    assertTrue(acceleratorTypes.iterator().hasNext());
+    assertNotNull(acceleratorTypes.iterator().next());
+  }
+
+  @Test
+  public void aggregatedListAcceleratorTypesTest() {
+    AggregatedListAcceleratorTypesPagedResponse aggregatedListAcceleratorTypes =
+        acceleratorTypeClient.aggregatedListAcceleratorTypes(PROJECT_NAME);
+    List<AcceleratorTypesScopedList> typesScopedLists =
+        Lists.newArrayList(aggregatedListAcceleratorTypes.iterateAll());
+    assertThat(typesScopedLists).isNotNull();
+    assertTrue(typesScopedLists.iterator().hasNext());
+    assertNotNull(typesScopedLists.iterator().next());
+  }
+
+  @Test
+  public void getAddressesTest() {
+    ProjectRegionAddressName projectRegionAddressName =
+        ProjectRegionAddressName.of(ADDRESS_NAME, DEFAULT_PROJECT, REGION);
+    Address address = addressClient.getAddress(projectRegionAddressName);
+    assertThat(address).isNotNull();
+    assertEquals(ADDRESS_TYPE, address.getAddressType());
+    assertEquals(ADDRESS_NAME, address.getName());
+    assertEquals(NETWORK_TIER, address.getNetworkTier());
+    assertEquals(REGION_LINK, address.getRegion());
+    assertEquals(ADDRESS_OPERATION_STATUS, address.getStatus());
+    assertEquals(ADDRESS_SELF_LINK, address.getSelfLink());
+  }
+
+  @Test
+  public void listAddressesTest() {
+    ListAddressesPagedResponse listAddresses = addressClient.listAddresses(PROJECT_REGION_NAME);
+    List<Address> addresses = Lists.newArrayList(listAddresses.iterateAll());
+    assertThat(addresses).isNotNull();
+    for (Address address : addresses) {
+      if (ADDRESS_NAME.equals(address.getName())) {
+        assertEquals(ADDRESS_TYPE, address.getAddressType());
+        assertEquals(ADDRESS_NAME, address.getName());
+        assertEquals(NETWORK_TIER, address.getNetworkTier());
+        assertEquals(REGION_LINK, address.getRegion());
+        assertEquals(ADDRESS_OPERATION_STATUS, address.getStatus());
+        assertEquals(ADDRESS_SELF_LINK, address.getSelfLink());
+      }
+    }
+  }
+
+  @Test
+  public void aggregatedListAddressesTest() {
+    AddressClient.AggregatedListAddressesPagedResponse pagedListResponse =
+        addressClient.aggregatedListAddresses(PROJECT_NAME);
+    List<AddressesScopedList> addressesScopedLists =
+        Lists.newArrayList(pagedListResponse.iterateAll());
```

Comment:
Here and everywhere else, apply the same

Reply:
done

Reply:
done